.github/CODEOWNERS

```diff
@@ -79,9 +79,12 @@
 /pkgs/development/tools/poetry2nix @adisbladis
 
 # Haskell
-/pkgs/development/compilers/ghc    @cdepillabout @sternenseemann @maralorn
-/pkgs/development/haskell-modules  @cdepillabout @sternenseemann @maralorn
-/pkgs/test/haskell                 @cdepillabout @sternenseemann @maralorn
+/doc/languages-frameworks/haskell.section.md  @cdepillabout @sternenseemann @maralorn
+/maintainers/scripts/haskell                  @cdepillabout @sternenseemann @maralorn
+/pkgs/development/compilers/ghc               @cdepillabout @sternenseemann @maralorn
+/pkgs/development/haskell-modules             @cdepillabout @sternenseemann @maralorn
+/pkgs/test/haskell                            @cdepillabout @sternenseemann @maralorn
+/pkgs/top-level/haskell-packages.nix          @cdepillabout @sternenseemann @maralorn
```

Comment:
The things that have been added are:

- `/doc/languages-frameworks/haskell.section.md`
- `/maintainers/scripts/haskell` (which was added in https://github.com/NixOS/nixpkgs/pull/121391)
- `/pkgs/top-level/haskell-packages.nix`

@sternenseemann @maralorn If either of you guys don't want to be in this list, let me know and I'll edit this PR.